frontend/src/components/ShootWorkers/WorkerGroup.vue

```diff
@@ -19,41 +19,194 @@ SPDX-License-Identifier: Apache-2.0
           {{workerGroup.name}}
       </v-chip>
     </template>
-    <v-list class="pa-0">
-      <v-list-item v-for="({title, value, description}) in workerGroupDescriptions" :key="title" class="px-0">
-        <v-list-item-content class="pt-1">
-          <v-list-item-subtitle>
-            {{title}}
-          </v-list-item-subtitle>
-          <v-list-item-title>{{value}} {{description}}</v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
-    </v-list>
+    <template v-slot:card>
+      <v-tabs
+        height="32"
+        color="primary"
+        v-model="tab">
+        <v-tab
+          key="overview"
+          href="#overview"
+        >
+          Overview
+        </v-tab>
+        <v-tab
+          key="yaml"
+          href="#yaml"
+        >
+          Yaml
+        </v-tab>
```

Comment:
```suggestion
        <v-tab
          key="overview"
          href="#overview"
          class="text-caption"
        >
          Overview
        </v-tab>
        <v-tab
          key="yaml"
          href="#yaml"
          class="text-caption"
        >
          Yaml
        </v-tab>
```
If we reduce the height we should also reduce the font-size.